### PR TITLE
Update to the Dremio Profile page

### PR DESCRIPTION
**Pull request by rheath-dremio**
## Description & motivation

I'm on the Docs team for Dremio, and I've updated the Dremio Profile page. We'd like the update to be published this Friday for Coalesce. Thanks!

### CODE REVIEW

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 1]
```diff
@@ -47,30 +47,257 @@ pip is the easiest way to install the adapter:
...
+When you initialize a project, you create one of these three profiles. You must configure it before trying to connect to Dremio Cloud or Dremio Software.
+
+* [Profile for Dremio Cloud](#cloud)
+* [Profile for Dremio Software with Username/Password Authentication](#software_up)
+* [Profile for Dremio Software with Authentication Through a Personal Access Token](#software_pat)
+
+For descriptions of the configurations in these profiles, see [Configurations](#configurations).
+
+<a id="cloud"></a>
+### Profile for Dremio Cloud
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/dremio-setup.md]: @rheath-dremio extra credit opportunity you want to turn the 3 profile options into tabs? Here's an example that uses Docusaurus's `Tab` and `File` elements.
https://github.com/dbt-labs/docs.getdbt.com/blob/a4399bfad96d209806de66332e7a69198caf8bb5/website/docs/reference/warehouse-setups/mssql-setup.md?plain=1#L103-L130

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 2]
```diff
@@ -47,30 +47,257 @@ pip is the easiest way to install the adapter:
...
+ <td><code>threads</code>
+ </td>
+ <td>Yes
+ </td>
+ <td><code>1</code>
+ </td>
+ <td>The number of threads the dbt project runs on.
+ </td>
+</tr>
+</table>
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/dremio-setup.md]: any chance you can convert these to markdown tables?

**Review by dataders — CHANGES_REQUESTED** [review]
I handled some merge conflicts -- looks like you made this PR off of a stale branch? I suggested just a few changes for readability.

File: website/docs/reference/warehouse-setups/dremio-setup.md (hunk 2 above)
Comment (rheath-dremio) [reply website/docs/reference/warehouse-setups/dremio-setup.md]: Absolutely. Done.

File: website/docs/reference/warehouse-setups/dremio-setup.md (hunk 1 above)
Comment (rheath-dremio) [reply website/docs/reference/warehouse-setups/dremio-setup.md]: Extra credit? I think I'm already an A student, but I'll take it. Done.

**Comment by dataders** [comment]
@rheath-dremio so one change you missed is that all the profile pages were renamed to setup pages. You mind making the changes to the `dremio-setup.md` and delete `dremio-profile.md`?

**Comment by rheath-dremio** [comment]
> @rheath-dremio so one change you missed is that all the profile pages were renamed to setup pages. You mind making the changes to the `dremio-setup.md` and delete `dremio-profile.md`?

Done. Thanks!

**Review by dataders — APPROVED** [review]
@rheath-dremio thanks for working together on this. I'm going to merge this now, but if you're ever interested, I think the Config sections could also be nicely tabbed as well. perhaps within in already defined profile tabs?

**Comment by rheath-dremio** [comment]
> @rheath-dremio thanks for working together on this. I'm going to merge this now, but if you're ever interested, I think the Config sections could also be nicely tabbed as well. perhaps within in already defined profile tabs?

Thank you, Anders, for all of your help! I'll circle back to this Dremio page after I get through some other projects, and reconfigure it.

**Pull request merged**